covbot.py

```diff
@@ -37,6 +37,11 @@
     'cases': ('!cases location', 'Get up to date info on cases, optionally in a specific location. You can give a country code, country name, state, country, region or city.'),
     'source': ('!source', 'Find out about my data sources and developers.'),
     'help': ('!help', 'Get a reminder what I can do for you.'),
+    'table': ('!table[html|short|tiny] location[s] ',
```

Comment:
Not sure how clear this is for end users?

There is also a case to be made for having multiple arguments so combinations are possible (eg `!table tiny html Loc1;Loc2;Loc3`), but I wanted to keep it as similar to `!cases` as possible

Reply:
Yeah, my preference would be to keep things simple